src/include/network/lan/complex.rb

```diff
@@ -149,10 +172,19 @@ def Commit
     def ReadDialog
       Wizard.RestoreHelp(Ops.get_string(@help, "read", ""))
       Lan.AbortFunction = -> { PollAbort() }
+
+      # FIXME: network-ng currently depends on this - do not read interfaces configuration
+      # itself yet
       ret = Lan.Read(:cache)
-      # Currently just a smoketest for new config storage - something what should replace Lan module in the bright future
-      # TODO: find a suitable place for this config storage
-      Y2Network::Config.from(:sysconfig)
+
+      # Load sysconfig into new backend
+      # It is done here bcs this method is currently called during "yast2 lan"
+      # initialization, so it makes configuration available for all submodules
+      system_config = Y2Network::Config.from(:sysconfig)
```

Comment:
Why here and not in `Lan.Read`?

Reply:
1) ```Lan``` module should be replaced by ```Y2Network::Config``` in the future
2) place it properly in ```Lan::Read``` is a bit tricky
3) I don't see any benefit from placing it into ```Lan::Read```